src/Symfony/Component/Validator/Resources/translations/validators.be.xlf

```diff
@@ -402,6 +402,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Значэнне сеткавай маскі павінна быць ад {{min}} да {{max}}.</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>Назва файла занадта доўгая. Ён павінен мець не больш за {{ filename_max_length }} сімвалаў.|Назва файла занадта доўгая. Ён павінен мець не больш за {{ filename_max_length }} сімвалаў.</target>
```

Comment:
`Назва файла` instead of just `файла`?

Also, is it normal that the plural version is exactly the same as the singular one? if yes, there is not need to duplicate.

Reply:
`Назва файла` is a filename. It's ok I think.
Regarding plural/singular I agree, there are differences.
But there are differences between x1, x2-3 and x4-x0, I see that RU language has 3 variants (and UK has only 2 same variants).

Reply:
Fixed, updated the translation to
`Назва файла занадта доўгая. Ён павінен мець {{ filename_max_length }} сімвал або менш.|Назва файла занадта доўгая. Ён павінен мець {{ filename_max_length }} сімвалы або менш.|Назва файла занадта доўгая. Ён павінен мець {{ filename_max_length }} сімвалаў або менш.`
(for numbers ending as x1, x2-x3, x4-x0)